### DISCUSSION TITLE
FolderPath configuration for StreamingFileLogger

### DISCUSSION THREAD

**Discussion by ataparia** [roubachof/MetroLog#29] [General]
Hi Team,
Is there any way to configure the FolderPath for StreamingFileLogger outside the MauiProgram. We have a scenario where our app is being used by multiple users and we would want to store the log file in folder's specific to the user which we create after the user logs in(i.e. after getting their user id). Just wanted to check if there is a way to configure the FolderPath outside of MauiProgram.cs.

**Comment by roubachof** [comment]
Hey!
I don't understand... If you are in a maui app, you can either use the standard configuration or the maui configuration.
If you use the maui configuration, which is in fact the ms logging extensions, you can inject `ILogger<XXXX> logger` in the constructors of your components.
If you use the standard configuration, you can use `LoggerFactory.GetLogger(nameof(XXXXX))`.
But you can't mix those.
Well you could use both, but that would be weird...
Please read the readme :)

If you want to set the logs path for the `StreamingFileTarget` with the standard configuration, there is an option for that in the constructor.

**Reply by ataparia** [reply]
Thanks @roubachof . I had to resort to using the manual configuration using LoggerFactory.Initialize(instance of LoggingConfiguration).
